File: P04 - Inherirance/Java/Mobil.java
package Praktikum.P04.P04c_Latihan_Inheritance_Java_227008_Elmosius_Suli;
// File : Mobil.java
// Nama : Elmosius Suli
// NRP  : 2272008
// Kelas : B
// Ket :
public class Mobil extends Kendaraan{
    public String bahanBakar;
    public String mesin;

    // constructor
    public Mobil(){
        this.bahanBakar = "Bensin";
        this.mesin = "EFI";
        super.kecepatanMaks = 100;
        super.kapasitasPenumpang = 4;
    }
}

File: P04 - Inherirance/Python/Mobil.py
from Kendaraan import Kendaraan

class Mobil(Kendaraan):
    def __init__(self) -> None:
        super().__init__()
        self.bahan_bakar: str = "Bensin"
        self.mesin: str = "EFI"
        self._kecepatan_maks= 100
        self._kapasistas_penumpang = 4